braindecode/models/eegconformer.py

```diff
@@ -0,0 +1,357 @@
+# Authors: Yonghao Song <eeyhsong@gmail.com>
+#
+# License: BSD (3-clause)
+
+import torch
+import torch.nn.functional as F
+from einops import rearrange
+from einops.layers.torch import Rearrange, Reduce
+from torch import nn, Tensor
+import warnings
+
+
+class EEGConformer(nn.Module):
+    """EEG Conformer.
+
+    Convolutional Transformer for EEG decoding.
+
+    The paper and original code with more details about the methodological
+    choices are available at the [Song2022]_ and [ConformerCode]_.
+
+    This neural network architecture recieves a traditional braindecode input.
+    The input shape should be three-dimensional matrix representing the EEG
+    signals.
+
+         `(batch_size, n_channels, n_timesteps)`.
+
+    The EEG Conformer architecture is composed of three modules:
+        - PatchEmbedding
+        - TransformerEncoder
+        - ClassificationHead
+
+    Notes
+    -----
+    The authors recommend using augment data before using Conformer, e.g. S&R,
+    at the end of the code.
```

Comment:
```suggestion
    The authors recommend using data augmentation before using Conformer, e.g. S&R,
    at the end of the code.
```
what is S&R? can you spell it out?